src/status_im/ui/components/toolbar.cljs

```diff
@@ -2,10 +2,11 @@
   (:require [status-im.ui.components.react :as react]
```

Comment:
Will create a file in `ui2`

Reply:
hmm, I thought we decided against using the ui or ui2 approach and there is a different project structure in place now.

@flexsurfer, @ilmotta where should code like this go. 

IMO it should be in src/status_im2/components/toolbar.cljs 
or if this is specific to messaging then it should be in

src/status_im2/messaging/components/toolbar.cljs 

Or is it the case that we are not quite ready for the code to be in the new structure yet? 🤔

Reply:
Sorry @J-Son89, it has been quite a challenge for me to review PRs in the midst of all refactorings + restructuring going on.

If something is not cristal clear, I'd say we should try to immediately update the new guidelines, there should be no doubt. But at the same time, I think we should try to freeze the guidelines as soon as possible and finish the migration first in order to avoid making the guidelines a moving target we can't reach.